docs/visualizing-your-data/snowplow-web/v1/index.md

```diff
@@ -0,0 +1,389 @@
+
+`Server / Database / Role / Schema`
+
+Once you made the changes you will need to refresh the data by clicking the *Refresh* icon on the *Home* ribbon when in the *Report view*.
+
+### Prefilter your data
+
+#### App id filter
+
+In case you would need to filter on a specific app_id, we have provided two parameters you can utilise for this purpose:
+
+`App_id_1 / App_id_2`
+
+You can of course add more, if needed, but these also need to be added to all queries, similar to the existing filters.
+
+#### Time filters
+
+The dashboard by design works with the most likely use case that it needs to display the last x number of days. You can change the `LastXDay` parameter to any of the default values (1, 7, 28, 30, 90) or add more, if you need to. This will ensure that the data is filtered for each of the visualizations.
+
+Lastly there are two more parameters, the `Start Year / End Year` which control the dimDate table, it should be set to cover a period you need for the duration of your reporting so you would only need to set it up once.
+
+Each of these parameters get updated once you refresh your database and are exposed in the Home page. There is a field called `Last Refreshed` which always show you the last time the queries were updated, which might come in handy for users to see.
+
+
+</TabItem>
+
+<TabItem value="Tableau" label="Tableau">
+
+### Prepare your Tableau workbook
+
+In order for you to be able to connect the workbook template to your dataset you will have to change the connection parameters in the template.
+
+First, unpackage the twbx file: change the twbx extension to zip and then you can treat it like one.
+
+Open the twb file. Use find and replace function in your editor of choice to replace all occurrences of the following connection parameters:
+
+`YOUR_SERVER, YOUR_WAREHOUSE, YOUR_DATABASE, YOUR_SCHEMA, YOUR_USER, YOUR_ROLE`
+
+Save the file.
+
+In order for Tableau to accept the twbx format you have to select all files first and then compress it back to a zip to avoid an additional level of folder structure. Once done rename the extension back to `twbx` from `zip` and your file should be ready to be used.
+
+### Prefilter your data
+
+Once the file is uploaded to Tableau online or opened elsewhere you will be prompted to log in with your username and password. Once authenticated you can change the data source settings e.g. switch the live connection to an extract.
+
+On the Home dashboard you will see a parameter called 'Last_x_day' which can be used to filter the number of days of data you would like the refresh to retrieve.
+
+
+</TabItem>
+
+<TabItem value="Streamlit" label="Streamlit">
+
+### Preparing the enviroment to run the application
+
+There are a few ways you can run the application:
+  1. [locally](#1-run-it-locally)
+  1. [Steamlit on Snowflake](#2-run-it-on-streamlit-on-snowflake)
+  1. [Streamlit cloud](#3-run-it-on-streamlits-cloud)
+
+<details>
+<summary>#### 1. Run it locally)</summary>
+
```

Comment:
Extra bracket here, also headers don't work in summary blocks (you'd need to use a h4 tag but they still won't show in the sidebar), I'd probably remove the header and hyperlinked stuff and just keep the collapsible block.